esmvaltool/utils/recipe_test_workflow/site/dkrz/env-file

```diff
@@ -22,6 +23,10 @@ export NUMEXPR_NUM_THREADS=1
 # Ensure '~/.local' isn't added to 'sys.path'.
 export PYTHONNOUSERSITE=True
 
+# Ignore 'ESMValCoreDeprecationWarning' warnings
+# related to the new user configuration file format.
+export PYTHONWARNINGS="ignore:Usage of the single configuration file::esmvalcore.config._config_object","ignore:The configuration option or command line argument::esmvalcore.config._config_validators"
```

Comment:
Since RTW is now using a config dir instead of a config_file, why is it getting deprecation warnings that we need to ignore?